xilem/examples/http_cats.rs

```diff
@@ -47,13 +47,15 @@ enum ImageState {
 
 impl HttpCats {
     fn view(&mut self) -> impl WidgetView<HttpCats> {
-        let left_column = portal(flex((
+        let left_column = sized_box(portal(flex((
             prose("Status"),
             self.statuses
                 .iter_mut()
                 .map(Status::list_view)
                 .collect::<Vec<_>>(),
-        )));
+        ))))
+        .padding(Padding::leading(20.));
```

Comment:
```suggestion
        .padding(Padding::leading(5.));
```

One thing to consider is that on Android, the amount of horizontal space is quite limited, which makes large paddings harder to justify (at least whilst we don't have a good dynamic layout). I also think that on desktop, a smaller padding here is nicer.

That being said, some padding here is definitely an improvement over none.

Reply:
I've accepted the change in the latest commit.